web-admin/src/components/home/DashboardList.svelte

```diff
@@ -13,30 +10,11 @@
 
   $: proj = createAdminServiceGetProject(organization, project);
   $: if ($proj.isSuccess && $proj.data?.productionDeployment) {
-    getDashboardsForProject($proj.data);
+    updateDashboardsForProject();
   }
 
-  async function getDashboardsForProject(projectData: V1GetProjectResponse) {
-    // Hack: in development, the runtime host is actually on port 8081
-    const runtimeHost = projectData.productionDeployment.runtimeHost.replace(
-      "localhost:9091",
-      "localhost:8081"
-    );
-
-    const axios = Axios.create({
-      baseURL: runtimeHost,
-      headers: {
-        Authorization: `Bearer ${projectData.jwt}`,
-      },
-    });
-
-    const { data } = await axios.get(
-      `/v1/instances/${projectData.productionDeployment.runtimeInstanceId}/catalog?type=OBJECT_TYPE_METRICS_VIEW`
-    );
-
-    dashboards = data.entries;
-
-    return;
+  async function updateDashboardsForProject() {
```

Comment:
This seems like a needless wrapper function.

Reply:
Needed since reactive statements dont support async-await. This is cleaner than writing `.then`